src/js/tabs/configuration.js

```diff
@@ -664,7 +438,7 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
             const MAX_DENOM = semver.gte(FC.CONFIG.apiVersion, "1.24.0") ? 16 : 8;
 
-            for (let denom = 1; denom <= MAX_DENOM; denom++) {
+            for (let denom = 1; denom < MAX_DENOM; denom++) {
```

Comment:
Is this actually fixing a bug, or is this accidental?

Reply:
There were problems with it during development (out of scope). Not anymore. Reverted.